packages/server/utils/getGraphQLExecutor.ts

```diff
@@ -1,14 +1,15 @@
 import {ExecutionResult} from 'graphql'
 import {ServerChannel} from 'parabol-client/types/constEnums'
+import SocketServerId from '../../client/shared/gqlIds/SocketServerId'
 import {GQLRequest} from '../graphql/executeGraphQL'
 import PubSubPromise from './PubSubPromise'
 
-let pubsub: PubSubPromise<GQLRequest & {serverChannel?: string}, ExecutionResult>
+let pubsub: PubSubPromise<GQLRequest & {executorServerId?: string}, ExecutionResult>
 const getGraphQLExecutor = () => {
   if (!pubsub) {
     pubsub = new PubSubPromise(
       ServerChannel.GQL_EXECUTOR_STREAM,
-      ServerChannel.GQL_EXECUTOR_RESPONSE
+      SocketServerId.join(process.env.SERVER_ID!)
```

Comment:
-1 Same as above, the result of this is a channel, not an id, so it should be either `...Id.split(...)` or `...Channel.join(...)`